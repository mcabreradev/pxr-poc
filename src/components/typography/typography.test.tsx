/* eslint-disable simple-import-sort/imports */
import 'jest-styled-components';

import { render, screen } from '@testing-library/react';
import Typography from './';

describe('Text component', () => {
  it('should renders Text component correctly', async () => {
    const text = 'Hola Paxer!';
    const renderComponent = () =>
      render(
        <Typography variant='base' weight='normal'>
          {text}
        </Typography>,
      );

    await renderComponent();
    const element = screen.getByText(text);
    // expect(element).toMatchSnapshot();
    expect(element).toBeInTheDocument();
  });
});
