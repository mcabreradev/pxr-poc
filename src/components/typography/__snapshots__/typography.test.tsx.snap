// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component should renders Text component correctly 1`] = `
<div
  className="font-normal text-base"
  style={{}}
  variant={6}
>
  Hello Paxer!
</div>
`;
